src/components/victory-bar/helper-methods.js

```diff
@@ -111,7 +111,10 @@ export default {
       );
 
       const text = this.getLabel(modifiedProps, datum, index);
-      const labelStyle = this.getLabelStyle(style.labels, datum);
+      const labelPadding = style.labels.padding;
+      const labelStyleObj = datum.y < 0 ? Object.assign({}, style.labels,
+        {padding: labelPadding - 7}) : style.labels;
```

Comment:
I believe you, but I would still be more comfortable if this number were expressed in terms of fontSize or padding or something less magical